tools/DfmHttpService/Handler/PreviewJsonConfig.cs

```diff
@@ -29,8 +29,11 @@ public class PreviewJsonConfig
         [JsonProperty("pageRefreshFunctionName")]
         public string PageRefreshFunctionName { get; set; }
 
-        [JsonProperty("port")]
-        public string Port { get; set; }
+        [JsonProperty("docfxServerPort")]
```

Comment:
yeah, serverPort is better.

---
In reply to: [108102911](https://github.com/dotnet/docfx/pull/1490#discussion_r108102911) [](ancestors = 108102911)